flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfiguration.java

```diff
@@ -54,7 +54,7 @@
 			@Nullable String jobClassName) {
 		super(configDir, dynamicProperties, args, hostname, restPort);
 		this.savepointRestoreSettings = requireNonNull(savepointRestoreSettings, "savepointRestoreSettings");
-		this.jobId = requireNonNull(jobId, "jobId");
```

Comment:
The annotation `@Nonnull JobID jobId` in the constructor parameter mismatches with new constrain.

Reply:
Done